helper_test.go

```diff
@@ -2439,6 +2439,13 @@ func genSafeRandomTerraformVersion() string {
 	return fmt.Sprintf("1.0.%d", rInt)
 }
 
+// genSafeRandomPolicyVersion returns a random version number of the form
+// `0.0.<RANDOM>`
+func genSafeRandomPolicyVersion() string {
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+	return fmt.Sprintf("0.0.%d", rInt)
+}
```

Comment:
Normally when we introduce a new resource to go-tfe, we should add a create helper to this file. `createAdminOpaVersion()`.